File: apps/institution-api/Models/InstitutionLocationWebsite.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class InstitutionLocationWebsite
    {
        public int InstitutionLocationWebsiteId { get; set; }
        public int? InstitutionLocationId { get; set; }
        public Guid WebsiteGuid { get; set; }
        public string WebsiteUsage { get; set; }
        public int? ListSequence { get; set; }
        public string WebsiteUrl { get; set; }
        public DateTime? WebsiteBusinessStartDate { get; set; }
        public DateTime? WebsiteBusinessEndDate { get; set; }

        public virtual InstitutionLocation InstitutionLocation { get; set; }
    }
}

File: libs/common/models/institution-location-website.ts


import { InstitutionLocation } from './institution-location';

export class InstitutionLocationWebsite {
    public institutionLocationWebsiteId: number = 0;
    public institutionLocationId: number;
    public websiteGuid: string = '';
    public websiteUsage: string = '';
    public listSequence: number;
    public websiteUrl: string = '';
    public websiteBusinessStartDate: Date;
    public websiteBusinessEndDate: Date;
    public institutionLocation: InstitutionLocation = new InstitutionLocation();

    constructor(init? : Partial<InstitutionLocationWebsite>) {
        
        Object.assign(this, init);
    }
}

